client/web/compose/src/components/ModuleFields/Editor/Select.vue

```diff
@@ -92,6 +105,14 @@
           </option>
         </template>
       </b-form-select>
+
+      <b-form-radio-group
+        v-else
```

Comment:
Do it the other way around, if its list view display that, otherwise(v-else) the default.